dali/kernels/common/scatter_gather.h

```diff
@@ -77,16 +77,23 @@ class DLL_PUBLIC ScatterGatherGPU {
     }
   }
 
+  enum class CopyMethod {
```

Comment:
That will cause excessively long, Java-style, qualified identifiers: `ScatterGather::CopyMethod::MemcpyAlways`...